### PR TITLE
Adjust kind clusters for 0.6.1

### PR DESCRIPTION
**Pull request by skitt**
This is a port of
https://github.com/submariner-io/submariner/commit/c2fc6b5e6db363d9ab555320827bf044b5c690e3
to submariner-operator, taking into account the new cluster naming
with recent versions of kind.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Comment by mangelajo** [comment]
> LGTM, but the deployment of lighthouse is failing :/

Yeah, I suspect may be lighthouse timeout is too low in travis? ...

File: scripts/kind-e2e/e2e.sh [hunk 1]
```diff
@@ -5,6 +5,21 @@ source $(git rev-parse --show-toplevel)/scripts/lib/debug_functions
 
 ### Functions ###
 
+function kind_fixup_config() {
```
Comment (dfarrell07) [inline scripts/kind-e2e/e2e.sh]: Maybe we can make these changes in submariner-io/submariner e2e.sh, then consume the fixes via https://github.com/submariner-io/submariner-operator/pull/248? I think the goal is to get rid of these scripts in submariner-operator ASAP. We could also try to get in https://github.com/submariner-io/submariner/pull/425 in first to take advantage of that refactoring.

**Comment by dfarrell07** [comment]
```
[scripts]$ kubectl get submariner submariner --namespace=submariner-operator -o jsonpath={.spec.serviceCIDR}
[scripts]$ grep 100.95.0.0/16
[submariner-operator]$ ./scripts/e2e keep true true
FATA[0996] exit status 1
```

File: scripts/kind-e2e/e2e.sh (hunk 1 above)
Comment (dfarrell07) [reply scripts/kind-e2e/e2e.sh]: Or second-thinking, maybe it's better just to fix it here for now and do this and other migrations when we move to shipyard.

**Comment by mkolesnik** [comment]
Closing since we moved to shipyard

**Pull request closed without merging**